aicsimageio/tests/metadata/test_utils.py

```diff
@@ -6,46 +6,39 @@
 
 from aicsimageio import AICSImage
 
-from ..conftest import get_resource_full_path, host
+from ..conftest import get_resource_full_path, LOCAL
 
 ###############################################################################
 
 
-@host
 @pytest.mark.parametrize(
     "filename",
     [
         # DefaultReader
         pytest.param(
             "example.png",
-            marks=pytest.mark.raises(execeptions=NotImplementedError),
+            marks=pytest.mark.raises(exception=NotImplementedError),
```

Comment:
It turns out these tests were falsely passing before due to a typo.

Reply:
nice catch, thank you !

Reply:
NICE